pages/vomnibar.coffee

```diff
@@ -222,7 +238,7 @@ class BackgroundCompleter
     id = BackgroundCompleter.messageId += 1
     @filterPort.onMessage.addListener handler = (msg) =>
```

Comment:
Why are we creating anonymous functions and messing with constantly adding and removing handlers, when we could just have one handler and ignore requests with `id != BackgroundCompleter.messageId`? Also makes this `keepAlive` complication redundant.